debian/control

```diff
@@ -28,6 +28,10 @@ Build-Depends: cdbs (>= 0.4.100), debhelper (>= 9), dpkg-dev (>= 1.16.1~),
  zlib1g-dev,
 # Install prove to run LuaJIT tests.
  libtest-harness-perl,
+# Install dependencies for functional tests.
+ python3-gevent,
+ python3-yaml,
+ tzdata,
```

Comment:
@ylobankov, @tsafin, can you clarify this dependency? Is it something new from recent timezones support in datetime? Is it just testing dependency or also a build dependency? Should we add it to the RPM spec as well?

Reply:
I don't see reading of any external file in those test cases. Whether `parse.date()` uses it? If so, it is not a testing dependency, but rather build or runtime one.

Reply:
> Is it something new from recent timezones support in datetime? Is it just testing dependency or also a build dependency? Should we add it to the RPM spec as well?

Yes, this is additional run-time dependency iff one asks for parsing of human-readable timezone names (e.g. 'Europe/Moscow') while parsing datetime literal, i.e. '2020-02-02T01:00 Europe/Moscow'. All other kinds of timezone offset (be them numeric, or abbreviations) do not add this extra runtime dependency.

P.S.
That was late night discovery before release of 2.10.0